.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: C++ CI Workflow
+
+on: [push]
+
+jobs:
+  build:
+    name: '[${{ matrix.os }}@${{ matrix.build_type }}]'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        build_type: [Debug, Release]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+
+    steps:
+    - uses: actions/checkout@master
+        
+    # Print environment variables to simplify development and debugging
+    - name: Environment Variables
+      shell: bash
+      run: env
+        
+    # ============
+    # DEPENDENCIES
+    # ============
+    
+    # Waiting a github action that installs dependencies in a
+    # multiplatform setup (likely using TS or JS), we install
+    # packages with os-specific steps.
+    
+    #- name: Dependencies [Ubuntu]
+    #  if: matrix.os == 'ubuntu-latest'
+    #  run: |
+    #    echo "Dependencies Ubuntu"
+
+    #- name: Dependencies [MacOS]
+    #  if: matrix.os == 'macOS-latest'
+    #  run: |
+    #    echo "Dependencies MacOS"
+
+    #- name: Dependencies [Windows]
+    #  if: matrix.os == 'windows-latest'
+    #  shell: bash
+    #  run: |
+    #    echo "Dependencies Windows"
+
+    # ===================
+    # CMAKE-BASED PROJECT
+    # ===================
+    
+    - name: Configure
+      # Use bash also on Windows (otherwise cd, mkdir, ... do not work)
+      shell: bash
+      run: |
+        cd $GITHUB_WORKSPACE/../how-to-export-cpp-library
+        mkdir -p build
+        cd build    
+        cmake -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} ..
+
+    - name: Build
+      shell: bash
+      run: |
+        cd $GITHUB_WORKSPACE/../how-to-export-cpp-library
+        cd build
+        cmake --build . --config ${{ matrix.build_type }}
+
+    - name: Test
+      shell: bash
+      run: |
+        cd $GITHUB_WORKSPACE/../how-to-export-cpp-library
```

Comment:
There is any way for making these `how-to-export-cpp-library` parametric in some way?

Reply:
Nah, it's a leftover. I used to clone this repo from another repo running this action, and I had to switch folder. It is not necessary in this case, the default initial directory of the steps is already that one.